src/coredata.cs

```diff
@@ -525,7 +571,7 @@ interface NSIncrementalStore {
 
 		[Static]
 		[Export ("identifierForNewStoreAtURL:")]
-		NSObject IdentifierForNewStoreAtURL (NSUrl storeURL);
+		NSObject IdentifierForNewStoreAtUrl (NSUrl storeUrl);
```

Comment:
I noticed this, did it in several places and I have wrapped all of them in XAMCORE_4_0